client/v3/op.go

```diff
@@ -554,7 +559,21 @@ func toLeaseTimeToLiveRequest(id LeaseID, opts ...LeaseOption) *pb.LeaseTimeToLi
 }
 
 // IsOptsWithPrefix returns true if WithPrefix option is called in the given opts.
-func IsOptsWithPrefix(opts []OpOption) bool { return isOpFuncCalled("WithPrefix", opts) }
+func IsOptsWithPrefix(opts []OpOption) bool {
+	ret := &Op{key: []byte("")}
```

Comment:
It would be cleaner to have `newOp()` factory method that takes care of setting the default versions of the fields/options.